tests/auditor_test.go

```diff
@@ -0,0 +1,86 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/MagalixCorp/magalix-agent/v3/agent"
+	"github.com/MagalixCorp/magalix-agent/v3/auditor"
+	"github.com/MagalixCorp/magalix-agent/v3/kuber"
+	"github.com/MagalixCorp/magalix-agent/v3/tests/mocks"
+	"github.com/MagalixTechnologies/uuid-go"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestConstraintsCache(t *testing.T) {
+	results := []*agent.AuditResult{}
+
+	entities := map[kuber.GroupVersionResourceKind][]unstructured.Unstructured{
```

Comment:
These are good for now, but we'll need to add entities with parents later so we can test the parent store dependent logic